src/System.Net.Primitives/tests/FunctionalTests/Resources/System.Net.Primitives.Functional.Tests.rd.xml

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Net.Primitives.Functional.Tests">
+    <Type Name="System.Net.CookieVariant" BinaryFormatter="All" />
+    <Type Name="System.Net.CookieCollection" BinaryFormatter="All" />
+  </Library>
+</Directives>
```

Comment:
Why are we making the changes here in the test library rd.xml file?  Shouldn't his be in the product source library System.Net.Primitives?  What does this rd.xml do here?

If we need to expose information for these specific types (CookieVariant, CookieCollection) for BinaryFormatter to work, this rd.xml file should be in the product library and not the test library.  Otherwise, all developers need to add this to their own rd.xml file.  Or that is the real goal here?  I.e., we want to force developers to have to do this?

cc: @danmosemsft @weshaggard @safern @ViktorHofer